packages/tasks_panel/src/commands/edit-task.ts

```diff
@@ -54,6 +41,8 @@ export async function editTask(task: ConfiguredTask, readResource: (file: string
       messages.DISCARD_CHANGES_BUTTON_TEXT()
     );
     if (decision !== messages.DISCARD_CHANGES_BUTTON_TEXT()) {
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion -- verified by condition above
+      void commands.executeCommand("tasks-explorer.tree.select", getTaskEditorPanel()!.getLoadedTask());
```

Comment:
How do we know for sure `getTaskEditorPanel()` is not undefined? It is not clear to me.